src/components/authentication-form/LoginForm.jsx

```diff
@@ -1,18 +1,97 @@
 import "./authentication.css";
+import { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
-import { PasswordInput } from "../password-input/PasswordInput";
+import { useAuth } from "../../contexts/auth/authContext";
+import { useNavigate } from "react-router-dom";
+import { Loader } from "../loader/Loader";
+import { useAxios } from "../../utils/useAxios";
+
 export const LoginForm = () => {
+  const navigate = useNavigate();
+  const [apiURL, setApiUrl] = useState(null);
+  const [showPwd, setShowPwd] = useState(false);
+  const { userAuthDispatch } = useAuth();
+  const [loginCredentials, setLoginCredentials] = useState({
+    email: "",
+    password: "",
+  });
+  const { email, password } = loginCredentials;
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    setApiUrl("/api/auth/login");
+  };
+  const { isLoaderLoading, serverResponse } = useAxios(apiURL, "POST", {
+    email: email,
+    password: password,
+  });
+  useEffect(() => {
+    const { data, status } = serverResponse;
+    if (status === 200) {
+      localStorage.setItem("token", JSON.stringify(data.encodedToken));
```

Comment:
You don't need to stringify the token here, as its already a string

Reply:
Yes got it thanks for pointing out.